app/converters/gbfs_donkey_kreuzlingen_set_current_range_meters.py

```diff
@@ -0,0 +1,35 @@
+from typing import List, Union
+
+from app.base_converter import BaseConverter
+
+
+class GbfsDonkeyKreuzlingenSetCurrentRangeMetersConverter(BaseConverter):
+    hostnames = ['stables.donkey.bike']
+
+    @staticmethod
+    def _get_system_id_from_path(path: str) -> str:
+        return path.split('/')[-3]
+
+    def convert(self, data: Union[dict, list], path: str) -> Union[dict, list]:
+        if not isinstance(data, dict):
+            return data
+
+        system_id = self._get_system_id_from_path(path)
+        if system_id != 'donkey_kreuzlingen':
+            return data
+
+        if path.endswith('/free_bike_status.json'):
+            fields = data.get('data', {})
+            if not isinstance(fields, dict):
+                return data
+            vehicles = fields.get('bikes', [])
+            if not isinstance(vehicles, list):
+                return data
+            for vehicle in vehicles:
+                if 'current_range_meters' in vehicle and vehicle['current_range_meters'] is None:
+                    vehicle['current_range_meters'] = 1000
```

Comment:
Where does `1000` come from? If donkey does not provide a `current_range_meters`, is `1000` a good value? Could you make this constant a bit more prominent by defining it explicitly and documenting it?
I suggest to also reference https://github.com/DonkeyRepublic/donkey_gbfs/issues/8 to keep track of any discussion regarding this issue.

Reply:
We used 1000 also for nextbike if current_range_meters is not defined. 1000 means: you still can ride, but not very far.